docs-starlight/src/styles/global.css

```diff
@@ -1,40 +1,347 @@
+  --sl-color-bg-inline-code: rgba(46, 42, 77, 0.8);
+  --sl-color-bg: #0f1731;
+
+	--sl-color-accent-low: #1f1d47;
+	--sl-color-accent: #5e46e6;
+	--sl-color-accent-high: #c0c3fa;
+	--sl-color-white: #ffffff;
+	--sl-color-gray-1: #e8eefc;
+	--sl-color-gray-2: #bbc2d4;
+	--sl-color-gray-3: #7e8bac;
+	--sl-color-gray-4: #4c5776;
+	--sl-color-gray-5: #2d3754;
+	--sl-color-gray-6: #1c2541;
+	--sl-color-black: #131824;
+
+  @media (width >= 768px) {
+    --sl-nav-height: 84px;
+  }
+}
+/* Docs Light mode colors. */
+:root[data-theme='light'] {
+  --sl-nav-height: 87px;
+  --sl-color-bg-nav: #0f1731;
+  --sl-color-bg-sidebar: #ffffff;
+  --sl-color-bg: #ffffff;
+
+	--sl-color-accent-low: #d0d3fc;
+	--sl-color-accent: #6049e8;
+	--sl-color-accent-high: #2c2669;
+	--sl-color-white: #131824;
+	--sl-color-gray-1: #1c2541;
+	--sl-color-gray-2: #2d3754;
+	--sl-color-gray-3: #4c5776;
+	--sl-color-gray-4: #7e8bac;
+	--sl-color-gray-5: #bbc2d4;
+	--sl-color-gray-6: #e8eefc;
+	--sl-color-gray-7: #f3f6fe;
+	--sl-color-black: #ffffff;
+
+  @media (width >= 768px) {
+    --sl-nav-height: 84px;
+  }
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 !important;
+}
+
+* {
+  box-sizing: border-box;
+}
+html, body {
+  overflow-x: hidden;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.no-scrollbar::-webkit-scrollbar {
+    display: none;
+}
+
+/* Hide scrollbar for IE, Edge and Firefox */
+.no-scrollbar {
+    -ms-overflow-style: none;  /* IE and Edge */
+    scrollbar-width: none;  /* Firefox */
+}
+
+.primary-button {
+  background-color: var(--color-accent-1);
+  border: 1px solid rgba(255, 255, 255, 0.2);
+  border-radius: 8px;
+  box-shadow: 0 0 15px 0 rgba(137, 107, 255, 0.30),
+              0 0 1px 4px rgba(52, 49, 27, 0.22),
+              0 0 1px 4px rgba(52, 49, 47, 0.22);
+  color: white;
+  font-family: var(--font-sans);
+  font-size: 14px;
+  font-weight: 400;
+  padding: 10px 24px;
+  text-decoration: none;
+}
+
+.secondary-button {
+  background-color: var(--color-bg-dark);
+  border: 1px solid var(--color-stroke-dark);
+  border-radius: 8px;
+  box-shadow: 0 0 15px 0 rgba(255, 255, 255, 0.05),
+              0 0 1px 4px rgba(52, 49, 27, 0.22);
+  color: white;
+  font-family: var(--font-sans);
+  font-size: 14px;
+  font-weight: 400;
+  padding: 10px 24px;
+  text-decoration: none;
+}
+
+.primary-button:hover, .secondary-button:hover {
+  cursor: pointer;
+}
+
+.primary-button:hover {
+  background-color: var(--color-accent);
+  transition: .25s ease-in-out;
+}
+
+.secondary-button:hover {
+  background-color: rgba(255, 255, 255, 0.05);
+  transition: .25s ease-in-out;
+}
+
+.primary-button:focus, .secondary-button:focus {
+  outline: 2px solid var(--color-accent-1);
+  outline-offset: 2px;
+}
+.primary-button:focus-visible, .secondary-button:focus-visible {
+  outline: 2px solid var(--color-accent-1);
+  outline-offset: 2px;
+}
```

Comment:
Any reason we're not putting these CSS styles into a more reusable button component?